app/services/lists/my_modules_service.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+module Lists
+  class MyModulesService < BaseService
+    PRELOAD = {
+      results: {},
+      my_module_status: {},
+      tags: {},
+      task_comments: { user: :teams },
+      user_assignments: :user,
+      designated_users: {},
+      experiment: :project
+    }
+
+    private
+
+    def fetch_records
+      records = @raw_data.includes(PRELOAD)
```

Comment:
all these method calls would probably be nicer if records was an instance variable @records, so we don't need to pass it everywhere